### ISSUE TITLE
⚡️ Easy issues for first-time contributors! 🍉

### ISSUE THREAD

**Issue by radex** [Nozbe/WatermelonDB#141]
Hey everyone! There’s a lot of you out there interested in using Watermelon! You can help us a lot in developing 🍉 by contributing to it — even in simple ways!

Here’s a list of easy fixes/improvements you can do to help:

- [ ] WIP

**Comment by lekterable** [comment]
Screw sleep, I was really curious about the code so I've did:

> Move arrayDifference from observation/fieldObserver to utils/fp/arrayDifference and add a couple of tests

Opening a PR for it in a minute.

**Comment by radex** [comment]
@lekterable Nice! Thank you for the suggestion — this list is written up definitely thanks to you

**Comment by cesargdm** [comment]
So talking about hooks, and taking advantage of its stable release. For example, ¿the way It should be to use hooks in observables could be?:

```javascript
import useObservable from '@nozbe/use-observable'

function Comment() {
  const { comment } = useObservables(['comment'], ({ comment }) => ({
    comment: comment.observe()
  }))
  
  return (  
    <div>
      <p>{comment.body}</p>
    </div>
  )
}

function Post() {
  const { post, comments } = useObservables(['post'], ({ post }) => ({
    post: post.observe(),
    comments: post.comments.observe()
  }))

  return (
    <article>
      <h1>{post.name}</h1>
      <p>{post.body}</p>
      <h2>Comments</h2>
      {comments.map(comment =>
        <EnhancedComment key={comment.id} comment={comment} />
      )}
    </article>
  )
}

```

**Comment by radex** [comment]
Yes! There's a proof of concept implementation of a useObservable on Nozbe/withObservables repo, but it hasn't been committed... A pull request would be nice

**Comment by co16353sidak** [comment]
Hey Everyone, I would love to work on updating the docs for watermelonDB regarding the `withObservables()` issue, would love it if I got the go ahead and a some tips on how to do this. 🥇

**Comment by radex** [comment]
@co16353sidak Hey, I _think_ the withObservables docs have been updated — but feel free to read through the docs carefully again and submit a PR with tweaks and fixes when you feel like something is not exactly understandable

**Comment by co16353sidak** [comment]
Is the issue regarding removing depreciated babel plugin still open?, would love to try this one out ?
@radex

**Comment by radex** [comment]
@co16353sidak go for it!

**Comment by saqibameen** [comment]
@radex I am interested in the first issue `Make Query and Relation thenable`. Can you give me some pointer to get started with it?

**Comment by lekterable** [comment]
Hey @radex , didn't know what would be the best way to reach you so I'll just post it here. Huge congrats on your talk on Chain React, I didn't attend it, but I've noticed your talk as It has been uploaded on (I think) one of the biggest youtube channels with tech-conf talks
https://www.youtube.com/watch?v=EFvywz7TQsA
You're a huge inspiration to me, keep it up! :))

**Comment by kriswep** [comment]
Working on updating the web demo to using `withDatabase` / `DatabaseContext`.

Will send a PR soon.
